tendrl/commons/flows/authorize_ssh_key/__init__.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+from tendrl.commons import flows
+from tendrl.commons.flows.exceptions import FlowExecutionFailedError
+from tendrl.commons.utils.ssh import authorize_key
+
+LOG = logging.getLogger(__name__)
+
+
+class AuthorizeSshKey(flows.BaseFlow):
+    def run(self):
+        ssh_key = self.parameters['ssh_key']
+        ret_val, err = authorize_key.AuthorizeKey(ssh_key).run()
+        if ret_val is not True or err != "":
+            raise FlowExecutionFailedError(err)
+        return True
+
+    def load_definition(self):
```

Comment:
You cannot override load_definition(), either mark this flow with class variable "internal=True" or define this flow correctly in definitions (.yml) file